### DISCUSSION TITLE
JetBrains is not a verified publisher

### DISCUSSION THREAD

**Discussion by DuncanCasteleyn** [JetBrains/Qodana#174] [Q&A] [answered]
Some companies require actions to be from verified creators, at the moment JetBrains is not verified.

Is this being requested? see https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#about-badges-in-github-marketplace

**Comment by tiulpin** [comment, marked as answer]
Thanks for asking. Yes, it is, should be done soon!

**Comment by tiulpin** [comment]
@DuncanCasteleyn we got it done and now the action is verified https://github.com/marketplace/actions/qodana-scan
